content/installation/bootstrapping.md

```diff
@@ -161,82 +201,199 @@ Getting your Cloudify Manager up and running comprises the following steps:
 
-### Bootstrap Validations
+An example output:
+{{< gsHighlight  sh  >}}
+$ cfy status
 
-During the first steps of the bootstrap process, validations take place. By default, if any of the validations fail, the bootstrap process also fails. The process validates such things as the volume of physical memory and disk space available on the host, that the relevant resources that are required for the bootstrap process are available for download, that supported OS distributions are being used for the Manager host, and so on.
+...
 
-To override validation preferences, see the `Bootstrap Validations` section in the `simple-manager-blueprint-inputs.yaml`.
+Retrieving manager services status... [ip=127.0.0.1]
 
-{{% gsNote title="Note" %}}
-Although it is possible ignore validations or change their defaults, it is not recommended that you do so without good reason.
-{{% /gsNote %}}
+Services:   
+   +--------------------------------+---------+
+   |            service             |  status |
+   +--------------------------------+---------+
+   | InfluxDB                       | running |
+   | Celery Management              | running |
+   | Logstash                       | running |
+   | RabbitMQ                       | running |
+   | AMQP InfluxDB                  | running |
+   | PostgreSQL                     | running |
+   | Manager Rest-Service           | running |
+   | Cloudify Stage                 | running |
+   | Webserver                      | running |
+   | Riemann                        | running |
+   | Webserver                      | running |
+   +--------------------------------+---------+
+
+   ...
+   {{< /gsHighlight >}}
 
+#### Step 5: Install Plugins
 
-## Offline Environment
+Install your required plugins. For more information, see [the Plugins section]({{< relref "plugins/using-plugins.md" >}}).
+
+#### Step 6: Configure Secret Storage
+
+Secret storage provides a tenant-wide store for data variables that you might not want to expose in plain text in Cloudify, such as login credentials for a platform. When you use secrets, the plugins that you have uploaded, consume the secrets to provide credential values. To implement secret storage for your tenants, see [_Using Secret Storage_]({{< relref "manager/using-secrets.md" >}}).
+
+
+## Installing Cloudify Manager in an Offline Environment
 
 This section describes how to bootstrap Cloudify Manager in an environment without an internet connection. 
 
-#### Process Overview
+When you are working offline in Cloudify, all resources required by Cloudify Manager, ranging from the bootstrap process to workflow execution, are contained within Cloudify Manager, rather than being retrieved from any other source, whether an internal network or (more commonly) a public network. Working offline provides advantages in the areas of stability and security and is a good solution for environments in which access to public networks is prohibited.
```

Comment:
Maybe we should rephrase this? " whether an internal network or (more commonly) a public network"